packages/react-topology/src/components/TopologyView/examples/Topology.md

```diff
@@ -1,155 +1,71 @@
-import { ItemDetails } from './ItemDetails';
-import { ViewToolbar } from './ViewToolbar';
-
-class SelectedTopologyView extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = { detailsShown: false };
-  }
-
-  render() {
-    const { detailsShown } = this.state;
-    const sideBar = <ItemDetails show={detailsShown} onClose={() => this.setState({ detailsShown: false })} />;
-    const controlButtons = createTopologyControlButtons({ fitToScreen: false, legend: false });
-
-    return (
-      <TopologyView
-        viewToolbar={<ViewToolbar />}
-        controlBar={<TopologyControlBar controlButtons={controlButtons} />}
-        sideBar={sideBar}
-        sideBarOpen={detailsShown}
-      >
-        <button onClick={() => this.setState({ detailsShown: !detailsShown })}>
-          {detailsShown ? 'Hide Details' : 'Show Details'}
-        </button>
-      </TopologyView>
-    );
-  }
-}
+### Baseline Topology
+
+1. Declare a controller, which can be initialized via the `useVisualizationController()` method.
+
+1. The `fromModel` method must be called on the controller to create the nodes. `fromModel` will take your data model as a parameter. Your data model should include a `graph` object, on which you will need to set `id` , `type` and `layout`.
+
+1. Create your topology view component, you can use `TopologyView` to Wrap `<VisualizationSurface>` which can accept `state` as a parameter. The state is application specific. It can be any data the application wants to store/retrieve from the controller. Adding state to the surface allows hooks to update when state changes. The state is useful to keep graph state such as selected elements.
+
+1. Use a controller to wrap your topology view component. In the example below, this is done via the `VisualizationProvider` which consumes the `Controller` via context.
+
+It is important to note that three `register` methods are accessed by the controller.
+
+The following two must be declared explicitly:
+
+- `registerLayoutFactory`: This method sets the layout of your topology view (e.g. Force, Dagre, Cola, etc.). You can use `defaultLayoutFactory` as a parameter if your application supports all layouts. You can also update `defaultLayout` to a custom implementation if you only want to support a subset of the available layout options.
+
+- `registerComponentFactory`: This method lets you customize the components in your topology view (e.g. nodes, groups, and edges). You can use `defaultComponentFactory` as a parameter.
+
+The register method below is initialized in `Visualization.ts`, therefore it doesn't need to be declared unless you want to support a custom implementation which modifies the types.
+
+- `registerElementFactory`: This method sets the types of the elements being used (e.g. graphs, nodes, edges). `defaultElementFactory` uses types from `ModelKind` and is exported in `index.ts`.
```

Comment:
The registration info should come direct after the first bullet `Create a controller...`